client/components/account-balances/test/index.test.tsx

```diff
@@ -33,6 +33,7 @@ const mockAccount: AccountOverview.Account = {
 		delay_days: 0,
 		interval: 'weekly',
 		weekly_anchor: 'Monday',
+		monthly_anchor: 1,
```

Comment:
In this PR I updated the `AccountOverview.Account` declaration to include the `monthly_anchor`. This change to the test is just to reflect that now required prop.